assets/scss/base/_table.scss

```diff
@@ -104,6 +104,11 @@ table {
     font-weight: 400;
   }
 }
+
+.table--heavy-header {
+    border-bottom: em(2) solid $govuk-blue-colour;
+}
```

Comment:
I suppose this is a broader question for designers what is the default border styling for a table?
The work on the `th` should be a reset in base. The component should have a standard border size.

```.table__header {
border-bottom: 2px solid;
border-color: <default>
}
```

Then we have a tones file
```
.tone-border-govuk {
border-colour: $govukBlue;
}
```

#### Implementation
```
<th class="table__header tone-border-govuk">
```

But that doesn't help the current situation!

> Is the default th styling not in effect the table__header? I would have thought that this is a variant of the default th? .table__header--heavy?

Potentially `.table__header--highlight` although this feels a little odd to me without a `.table__header` class. You get the table styles for free without a selector name applied, apply `.table__header` and you get the custom styles.

Or following the same patten then:

```
.table--header-highlight {
   th {
      border-bottom: em(2) solid $govuk-blue-colour;
   }
}
```